src/test/resources/application.properties

```diff
@@ -0,0 +1,8 @@
+spring.h2.console.enabled=true
+
+spring.datasource.url=jdbc:h2:mem:testdb;DB_CLOSE_DELAY=-1
```

Comment:
```suggestion
spring.datasource.url=jdbc:h2:mem:testdb;
```

Reply:
We use it, for tests. We can't remove it